### PR TITLE
Cpp subscriber

### PR DESCRIPTION
**Pull request by isaiahb**

### CODE REVIEW

File: cpp_subscriber/message.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+UR p(�	0�
```
Comment (viktries-google) [inline cpp_subscriber/message.txt:1]: what is this file? is this needed?

File: cpp_subscriber/processor.h [hunk 2]
```diff
@@ -48,7 +52,10 @@ google::protobuf::util::StatusOr<ConfigChangeRequest> MessageProcessor(Message c
   using google::protobuf::util::StatusOr;
   using google::protobuf::util::Status;
   using google::protobuf::util::error::Code;
-
+  using youtube_hermes_config_subscriber::PublishMessage;
+  using youtube_hermes_config_subscriber::getDummyImpactAnalysis;
```
Comment (viktries-google) [inline cpp_subscriber/processor.h:56]: these aren't needed. you're already in this namespace

File: cpp_subscriber/publisher.h [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+#include "google/pubsub/v1/pubsub.grpc.pb.h"
+#include "proto/config_change.pb.h"
+#include "google/pubsub/v1/pubsub.grpc.pb.h"
+#include "absl/strings/string_view.h"
+
+namespace youtube_hermes_config_subscriber {
+
+// Generates and returns a serialized Impact Analysis Response containing queue 
+std::string getDummyImpactAnalysis(const ConfigChangeRequest);
```
Comment (viktries-google) [inline cpp_subscriber/publisher.h:34]: you aren't mutating these, why make a copy?

const &ConfigChangeRequest config_change_request

File: cpp_subscriber/message.txt (hunk 1 above)
Comment (isaiahb) [reply cpp_subscriber/message.txt:1]: it was a serialized proto object, this file is no longer needed.

**Pull request merged**